src/string/modifiers.ts

```diff
@@ -0,0 +1,4 @@
+export const stringToArray = (str = '', delimiter = ','): string[] => {
+  if (typeof str !== 'string' || str === '') return []
+  return decodeURIComponent(str).trim().split(delimiter)
```

Comment:
Can u remove this decodeURIComponent? if I try to get an array of the code list 'abçl21#%5a;khc2#%1a%' it will be break my code :/

Reply:
It is need.
Lets say we wanna grab the numbers presents on a data that is a pathParameter from a lambda...  date/{date}.

Reply:
@mickaelnn

Reply:
the function is, stringToArray, not decodedURIStringToArray :/

Reply:
A good practice would be to decode the string first and then use this new string as the function first parameter. This way the function will be handful in a lot of scenarios.